server/helpers/utils.js

```diff
@@ -0,0 +1,183 @@
+import jwt from 'jsonwebtoken';
+import bcrypt from 'bcrypt';
+import { Users } from '../models';
+
+/**
+ * @description hash user password using bcrypt
+ * @function passwordHash
+ * @param {string} password user supplied password
+ * @returns {string} hashed user password
+ */
+export const passwordHash = password => bcrypt.hashSync(password, 10);
+
+/**
+ * @description hash user password using bcrypt
+ * @function isRegisteredUser
+ * @param {int} userIdentifier attribute to identify user
+ * @param {string} type field we are using to query
+ * @returns {boolean} boolean of registered user status
+ */
+export const isRegisteredUser = (userIdentifier, type = 'id') => {
+  const queryWhere = (type === 'id') ? { userId: userIdentifier } : { email: userIdentifier };
+  return Users.findOne({
+    where: queryWhere,
+    plain: true,
+  })
+  .then((returnedUser) => {
+    const user = returnedUser.get({ plain: true });
+    if (user) {
```

Comment:
This is repittion at best why don't use just return a user. if there is a user its truthy. if there isn't it is falsey